### ISSUE TITLE
Build a complete worked example (router rebooter at 03:00 nightly for 1 minute)

### ISSUE THREAD

**Issue by jonathangroves** [whaleygeek/pyenergenie#110]
This would be a great use case for the Energenie range controlled by a Pi. An "If This, Then That" rule would fail due to the loss of connectivity when the router goes off but of course a Pi uses radio to communicate with the Energenie device so connectivity over IP is irrelevant. Many home routers need regular reboots and being able to do this from the Pi at a pre-determined time would be very useful.

**Comment by whaleygeek** [comment]
Good idea! Leave this issue open and we can add a demo to the standard demo sources at some time in the future.
